persistence/src/main/java/de/cyface/persistence/MeasurementTable.java

```diff
@@ -73,35 +77,72 @@ protected String getCreateStatement() {
         return "CREATE TABLE " + getName() + " (" + BaseColumns._ID + " INTEGER PRIMARY KEY AUTOINCREMENT, "
                 + COLUMN_STATUS + " TEXT NOT NULL, " + COLUMN_VEHICLE + " TEXT NOT NULL, " + COLUMN_ACCELERATIONS
                 + " INTEGER NOT NULL, " + COLUMN_ROTATIONS + " INTEGER NOT NULL, " + COLUMN_DIRECTIONS
-                + " INTEGER NOT NULL, " + COLUMN_PERSISTENCE_FILE_FORMAT_VERSION + " SHORT INTEGER NOT NULL, "
```

Comment:
seems like there is no "short integer" type in SQLite as IntelliJ showed my an error and I did not find it in the documentation ... I wonder were I found this when I added this ^^

Reply:
SQLite however automatically uses INT for all types which contains the chars "INT" ^